base/linalg/arpack.jl

```diff
@@ -228,7 +228,7 @@ for (T, saupd_name, seupd_name, naupd_name, neupd_name) in
                   sizeof(howmny), sizeof(bmat), sizeof(evtype))
         end
 
-        function saupd(ido, bmat, n, which, nev, TOL::Array{$T}, resid::Array{$T}, ncv, v::Array{$T}, ldv,
+        function saupd(ido, bmat, n, which, nev, TOL::Ref{$T}, resid::Array{$T}, ncv, v::Array{$T}, ldv,
```

Comment:
I think we'll need to revert the changes that were visible in the signature here, this broke MatrixNetworks.jl https://github.com/JuliaCI/pkg.julialang.org/blob/c868b6c9a62315ab23df2331ffd159c23cb29f2e/logs/MatrixNetworks_0.6.log and we're in feature freeze